trace/stats.go

```diff
@@ -15,6 +15,9 @@ import (
 type PacketCountConsumer interface {
 	// Add an additional measurement to the current count
 	Update(delta PacketCounts)
+
+	// Get the current total counts
+	Get() PacketCounts
```

Comment:
I don't like this at all.  Could the trigger be moved into PacketCounter, instead of requiring a round-trip through the interface?

Then it would not have to be duplicated in each place, either?

Reply:
PacketCounter is used in multiple summaries, not just the filter summary. I think the parent, which is PacketCounterCollector is the correct place to do it.